StravaDiscordBot/Models/Categories/VirtualRideCategory.cs

```diff
@@ -10,6 +10,7 @@ namespace StravaDiscordBot.Models.Categories
     public class VirtualRideCategory : ICategory
     {
         public string Name => Constants.CategoryTypes.VirtualRide;
+        public const string IndoorRideName = "Indoor Cycling";
```

Comment:
This is not really a limitation on wahoo's side. Virtual Ride category requires GPS coordinates. There is no reasonable way to have GPS data for a stationary bike and such. Wahoo app instead categorises it as a Ride with indoor tag, because that is what Strava imposes. Problem is, I don't see any Tag property within the activity, so I can't find another way to search for indoor ride.